frontend/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM oven/bun@sha256:5ba9a68692e9fd7aa435a79b2597f8211fbef2c6b0bed0ce9c9293c98fef5848
+# Use the official Bun image (using a tag version for consistency)
+FROM oven/bun:1 AS base
```

Comment:
Do we think renovatebot will tolerate using numbered version tags?  It historically has not in favor of pinned versions, which is more secure.

Reply:
will test and see if the sha version works with this solution

Reply:
app did not run at all using the SHA option so will revert back and merge